Steps4Impact/Settings/TeamSettingsViewController.swift

```diff
@@ -71,11 +71,34 @@ extension TeamSettingsViewController: SettingsActionCellDelegate {
           viewController: self,
           shareButton: button,
           string: Strings.Share.item)
+    case .editname:
+      AKFCausesService.getParticipant(fbid: Facebook.id) { (result) in
+        guard let participant = Participant(json: result.response) else { return }
+        guard participant.team != nil else { return }
+        let alert = UIAlertController(title: Strings.TeamSettings.editTeamName,
+                                      message: Strings.TeamSettings.editTeamNameMessage,
+                                      preferredStyle: UIAlertController.Style.alert)
+        let action = UIAlertAction(title: Strings.TeamSettings.update, style: .default) { (_) in
```

Comment:
Not sure if it helps outlining this.  Why not just put this inline in the `addAction`?